README.md

```diff
@@ -302,13 +302,15 @@ Hurray! You successfully made a contribution!
   - [Jayvir Rathi](https://github.com/Jayvirrathi)
   - [Jessica Salbert](https://github.com/jessicasalbert)
   - [Jivthesh](https://github.com/jivthesh)
+  - [John Daniel](https://github.com/johndanny98)
   - [Johanna](https://github.com/Johanna-hub)
   - [Jome Favourite](https://github.com/jomefavourite)
   - [Jonathan Niklasson Godar](https://github.com/JonathanGodar)
   - [Jose Angel Munoz](https://github.com/imjoseangel)
   - [Joseph Jose](https://github.com/josephjosedev)
   - [Josh Callahan](https://github.com/joshcallahan)
   - [Joykishan Sharma](https://github.com/JoykishanSharma)
+  
```

Comment:
```suggestion
```

Reply:
@Panquesito7  had removed the space and name on line 306